src/api/workqueue/workqueueConfig.ts

```diff
@@ -74,7 +79,270 @@ export const Workqueues = defineWorkqueue([
     ],
     actions: [],
     query: {
-      status: { type: 'exact', term: 'WAITING_FOR_ATTESTATION' }
+      type: 'and',
+      clauses: []
     }
+  },
+
+  {
+    slug: 'assigned-to-you',
+    icon: 'PushPin',
+    name: {
+      id: 'workqueues.assignedToYou.title',
+      defaultMessage: 'Assigned to you',
+      description: 'Title of assigned to you workqueue'
+    },
+    query: {
+      type: 'and',
+      clauses: [
+        {
+          assignedTo: { type: 'exact', term: user('id') }
+        }
+      ]
+    },
+    actions: [
+      {
+        type: 'REVIEW',
+        conditionals: []
+      }
+    ]
+  },
+  {
+    slug: 'recent',
+    icon: 'Timer',
+    name: {
+      id: 'workqueues.recent.title',
+      defaultMessage: 'Recent',
+      description: 'Title of recent workqueue'
+    },
+    query: {
+      type: 'and',
+      clauses: [
+        {
+          updatedBy: { type: 'exact', term: user('id') },
+          updatedAt: {
+            type: 'range',
+            gte: new Date(Date.now() - 1000 * 60 * 60 * 24 * 7).toISOString(),
+            lte: new Date(Date.now()).toISOString()
+          }
+        }
+      ]
+    },
+    actions: [
+      {
+        type: 'REVIEW',
+        conditionals: []
+      }
+    ]
+  },
+  {
+    slug: 'requires-completion',
+    icon: 'File',
+    name: {
+      id: 'workqueues.notifications.title',
+      defaultMessage: 'Notifications',
+      description: 'Title of notifications workqueue'
+    },
+    query: {
+      type: 'and',
+      clauses: [
+        {
+          status: { type: 'exact', term: 'NOTIFIED' }
+        }
+      ]
+    },
```

Comment:
Can you add a piece of logic inside `defineWorkqueues` to convert 

```
{
  status: { type: 'exact', term: 'NOTIFIED' }
}
```

into

```
{
    type: 'and',
    clauses: [
      {
        status: { type: 'exact', term: 'NOTIFIED' }
      }
    ]
  }
```

Just so we don't have to ever answer the "why do I need to add the 'and' wrapper even with one clause"